instrumentation/net/http/otelhttp/common.go

```diff
@@ -31,10 +31,16 @@ const (
 
 // Server HTTP metrics.
 const (
-	RequestCount          = "http.server.request_count"           // Incoming request count total
-	RequestContentLength  = "http.server.request_content_length"  // Incoming request bytes total
-	ResponseContentLength = "http.server.response_content_length" // Incoming response bytes total
-	ServerLatency         = "http.server.duration"                // Incoming end to end duration, milliseconds
+	ServerRequestSize  = "http.server.request.size"  // Incoming request bytes total
+	ServerResponseSize = "http.server.response.size" // Incoming response bytes total
+	ServerDuration     = "http.server.duration"      // Incoming end to end duration, milliseconds
+)
+
+// Client HTTP metrics.
+const (
+	ClientRequestSize  = "http.client.request.size"  // Outgoing request bytes total
+	ClientResponseSize = "http.client.response.size" // Outgoing response bytes total
+	ClientDuration     = "http.client.duration"      // Outgoing end to end duration, milliseconds
```

Comment:
I guess it is not aligned with semconv from OTEL: https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-client